pandas/tests/series/test_constructors.py

```diff
@@ -28,6 +28,30 @@
 
 class TestSeriesConstructors():
 
+    @pytest.mark.parametrize('constructor,check_index_type', [
+        # NOTE: some overlap with test_constructor_empty but that test does not
```

Comment:
Rather than duplicating is there a logical way to replace it? The amount of parametrization here is more comprehensive so would make sense to leverage

Reply:
this seems ok here. ok with some duplication if its easier to read / grok.